build/scripts/make-version.sh

```diff
@@ -11,15 +11,19 @@ set -o errexit
 function get_version() {
```

Comment:
add comment for input and expect result pls.

Reply:
added in top of script